### ISSUE TITLE
weights lambda

### ISSUE THREAD

**Issue by DirtyHarryLYL** [CQFIO/PhotographicImageSynthesis#10]
Thx for the great project first~
My question:
In the train demo,what does " the weights lambda are collected at 100th epoch" mean?
Should I change lambda in  "tf.image.resize_area(label,(sp//16,sp//8)))*10/1.5" with new value got in training by hand? 
Sincerely

**Comment by DirtyHarryLYL** [comment]
Another 2 question:
1. In demo_256p.py, why the "G_loss=tf.reduce_sum(tf.reduce_min(content_loss,reduction_indices=0))*0.999+tf.reduce_sum(tf.reduce_mean(content_loss,reduction_indices=0))*0.001", 
but in other demos, the G_loss is directly given "p0+p1+p2+p3+p4+p5"?
2. In the helper.py, function "get_semantic_map" need  a json file to convert: "tmp[:,:,k]=np.float32((semantic[:,:,0]==dataset.palette[k,0])&(semantic[:,:,1]==dataset.palette[k,1])&(semantic[:,:,2]==dataset.palette[k,2]))", 
could you please tell me what's the format of this json file or where to find it? Is it in cityscapes dataset?
Thx~

**Comment by CQFIO** [comment]
For the first question, the weights are collected by taking the inverse of the average of l0,..,l5 in the 100th epoch. For the second question, in demo_256p, I use the diversity loss and I use the simple loss in demo_512p and demo_1024p. I generate 9 output images in demo_256p.

**Comment by DirtyHarryLYL** [comment]
Thanks a lot~
p_i=compute_error(~~)*mean(l_i)_100th_epoch, i=0,1,...5
